iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/compaction/schedule/CompactionTaskManager.java

```diff
@@ -84,17 +84,19 @@ public class CompactionTaskManager implements IService {
 
   private final RateLimiter mergeWriteRateLimiter =
       RateLimiter.create(
-          IoTDBDescriptor.getInstance().getConfig().getCompactionWriteThroughputMbPerSec()
-              * 1024.0
-              * 1024.0);
+          config.getCompactionWriteThroughputMbPerSec() <= 0
+              ? Double.MAX_VALUE
+              : config.getCompactionWriteThroughputMbPerSec());
```

Comment:
Should the value not be multiplied by 1024*1024?
I assume its unit is MB while the RateLimiter's is Byte.

Reply:
It should be multiplied by 1024 * 1024. I forgot the unit in previous modification.